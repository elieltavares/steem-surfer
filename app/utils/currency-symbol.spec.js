/* eslint-disable */
import currencySymbol from './currency-symbol';

describe('currency-symbol', () => {
  it('(1) usd', () => {
    const input = 'usd';
    expect(currencySymbol(input)).toMatchSnapshot();
  });

  it('(2) eur', () => {
    const input = 'eur';
    expect(currencySymbol(input)).toMatchSnapshot();
  });

  it('(3) try', () => {
    const input = 'try';
    expect(currencySymbol(input)).toMatchSnapshot();
  });
});
