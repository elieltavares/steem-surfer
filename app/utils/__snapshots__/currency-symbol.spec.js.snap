// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`currency-symbol (1) usd 1`] = `"$"`;

exports[`currency-symbol (2) eur 1`] = `"€"`;

exports[`currency-symbol (3) try 1`] = `"₺"`;
